src/utils/constant/images.js

```diff
@@ -1,8 +1,11 @@
 import logo from "assets/images/icons/logo.png";
 import search from "assets/images/icons/search.png";
+import deleteIcon from "assets/images/icons/deleteIcon.png";
 
 export const LOGO = logo;
 export const SEARCH = search;
+export const DELETE = deleteIcon;
```

Comment:
use common prefix to group similar constants together.

```suggestion
export const ICON_DELETE = deleteIcon;
```